static/docs/commands-reference/remote_add.md

```diff
@@ -252,6 +252,57 @@ variables:
 
 </details>
 
+<details>
+
+### Click for Aliyun OSS
+
+First you need to setup OSS storage on Aliyun Cloud and then 
+use s3 style url for oss storage and make endpoint a configurable value, 
+an example is shown below:
+
+```dvc
+$ dvc remote add myremote oss://my-bucket/path
+```
+
+To set key id, key secret and endpoint you need to use modify command from dvc, 
+a sample ussge is show below:
+
+```dvc
+$ dvc remote modify myremote oss_key_id my-key-id --local
+$ dvc remote modify myremote oss_key_secret my-key-secret --local
+$ dvc remote modify myremote oss_endpoint endpoint --local
```

Comment:
1. let's use `dvc remote modify myremote --local ...` it's easier to read
2. no need for `--local` for the endpoint one

Reply:
@shcheklein updated positions of flags in my latest commits.